### ISSUE TITLE
classify fails with "alias" and "status"

### ISSUE THREAD

**Issue by solnic** [dry-rb/dry-inflector#27]
```ruby
[14] pry(main)> i.classify('status')
=> "Statu"
[15] pry(main)> i.classify('statuses')
=> "Status"
[16] pry(main)> i.classify('alias')
=> "Alia"
[16] pry(main)> i.classify('aliases')
=> "Alias"
```

refs rom-rb/rom#564

**Comment by Ptico** [comment]
I’ll be happy to pair on this any time next week

**Comment by flash-gordon** [comment]
Closed per #28
